news/2023/2023-01-25-new-featured-artist-winterhorde.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "New Featured Artist: WINTERHORDE"
+date: 2023-01-25 11:00:00 +0000
+---
+
+Melodic black/death metal band **WINTERHORDE** takes a step into our library of Featured Artist.
+
+![](https://assets.ppy.sh/artists/337/header.jpg)
+
+**WINTERHORDE**'s music is everything a metalhead could ask for: high-tempo, high-intensity, and abundantly dense with drums. Stream-focused metal mapping (which should really be called *blast beatmapping*) is their speciality, as made crystal clear by maps like [Chronic Death](https://osu.ppy.sh/beatmapsets/1699956) and [Worms of Soul](https://osu.ppy.sh/beatmapsets/1724057).
+
+Wrench your mind around the **6** tracks we've picked up for [**WINTERHORDE**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/337), all pre-timed and ready for your Ctrl+Shift+F mastery.
+
+Experience **WINTERHORDE**'s sound for yourself:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/337/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### WINTERHORDE - Worms of Soul
+
+Check out [the map from the video above](https://osu.ppy.sh/beatmapsets/1724057) hosted by [kiwies](https://osu.ppy.sh/users/12927633)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/337/Maestro/Winterhorde%20-%20Worms%20of%20Soul.mp3" type="audio/mpeg">
+</audio>
+
+### WINTERHORDE - Chronic Death
+
+Or try [this other ultra-difficult marathon](https://osu.ppy.sh/beatmapsets/1699956) hosted by [iamtickrate](https://osu.ppy.sh/users/10772580)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/337/Maestro/Winterhorde%20-%20Chronic%20Death.mp3" type="audio/mpeg">
+</audio>
+
+### WINTERHORDE - The Greatest Plague of Earth
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/337/The%20Greatest%20Plague%20of%20Earth%2FWinterhorde%20-%20The%20Greatest%20Plague%20of%20Earth.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+[**WINTERHORDE**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/337) has even more songs for you to smash your keyboard along to. Pick whichever suits your fancy and design your own finger-demolishing canvas of chaos (also known as a beatmap).
+
+Our next Featured Artist might be a day late for... undisclosed reasons. Keep an eye on the [@osugame Twitter](https://twitter.com/osugame) for when it drops, and maybe watch this weekend's [MWC7K](/wiki/Tournaments/MWC/2023_7K) for... similar undisclosed reasons.
```

Comment:
```suggestion
Our next Featured Artist might be a day late for... undisclosed reasons. Keep an eye on the [@osugame Twitter](https://twitter.com/osugame) for when it drops, and maybe watch this weekend's [MWC 7K](/wiki/Tournaments/MWC/2023_7K) for... similar undisclosed reasons.
```

Reply:
(it's spelled this way according to the wiki)

![image](https://user-images.githubusercontent.com/36564236/214530965-71320228-f527-4699-86fe-93a042aecf4e.png)